### ISSUE TITLE
Cannot find source file foo.ts

### ISSUE THREAD

**Issue by xadhoom** [s-panferov/awesome-typescript-loader#163]
Hi,

I'm using the primeng npm module but the loader warns about not being able to find any foo.ts, for example:  

WARNING in ./~/primeng/primeng.js
Cannot find source file 'primeng.ts': Error: Cannot resolve 'file' or 'directory' ./primeng.ts in /home/matteo/devel/detectino/apps/dt_web/webapp/node_modules/primeng

The npm package has no foo.ts, but only foo.d.ts, then foo.js and foo.js.map

The warning does not seems to cause any issue.

Can be a primeng issue or something I'm missing?

(I'm using a simplified version of https://github.com/AngularClass/angular2-webpack-starter for my test project)

**Comment by s-panferov** [comment]
@xadhoom  hello,

right now I'm working on the next version of the loader. Could you please try it (`2.0.0-rc.3` right now)? These errors must be solved there.

**Comment by xadhoom** [comment]
I've tried rc.10, but same errors.

In addition, it complains with rxjs (I'm also using angular2 latest release) with errors like:
Invalid module name in augmentation, module '../../Observable' cannot be found.

**Comment by s-panferov** [comment]
```
Invalid module name in augmentation, module '../../Observable' cannot be found.
```

This looks like a TS bug, see [my another comment](https://github.com/s-panferov/awesome-typescript-loader/issues/168#issuecomment-227359572)

**Comment by s-panferov** [comment]
Fixed in latest `2.0.0` release candidates.
